Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -69,7 +69,7 @@ protected function getClassMetadata(ResourceInterface $resource)
      */
     protected function isPropertyEnabled($property)
     {
-        return null === $this->properties || isset($this->properties[$property]);
+        return null === $this->properties || array_key_exists($property, $this->properties) && false !== $this->properties[$property];
```

Comment:
false is a valid value here. The property is enabled if the key exist (or if $this->properties is null):

```php
return null === $this->properties || array_key_exists($property, $this->properties);
```